scripts/build/Jenkins/Jenkinsfile

```diff
@@ -76,11 +76,11 @@ def palRm(path) {
 def palRmDir(path) {
     if (env.IS_UNIX) {
         sh label: "Removing ${path}",
-           script: "rm -rf ${path}"
+           script: "if [ -d ${path} ]; then rm -rf ${path}; fi"
```

Comment:
Is this check required? command won't raise an error if directory doesn't exist.

Reply:
When I tested it, the command was raising an error so that's why I added this check